src/templates/article.js

```diff
@@ -53,101 +220,374 @@ const PreviewContainer = styled.div`
+            font-size: 14px;
+        }
+        & ul {
+            > li {
+                font-size: 14px;
+                line-height: 20px;
+            }
+        }
+        & h1 {
+            font-size: 32px;
+            line-height: 40px;
+        }
+        & h2 {
+            font-size: 28px;
+            line-height: 34px;
+        }
+        & h3 {
+            font-size: 24px;
+            line-height: 30px;
+
+            & + p {
+                margin-top: 8px;
+            }
+        }
+        & h4 {
+            font-size: 18px;
+            line-height: 26px;
+
+            & + p {
+                margin-top: 8px;
+            }
+        }
+        & h5 {
+            font-size: 16px;
+            line-height: 24px;
+        }
+        & h6 {
+            font-size: 14px;
+            line-height: 20px;
+        }
+    }
+`
+const SocialComponentsWrapper = styled(Flex)`
+    justify-content: space-between;
+    border-top: 1px solid var(--color-grey-6);
+    padding-top: 24px;
+    margin: 40px 0;
+`
+const LeftSocialComponents = styled.div`
+    width: 10px;
+`
+const RightSocialComponents = styled.div`
+    width: auto;
 `
 
-const ArticleTemplate = (props) => {
-    const post_data = props.data.directus.articles[0]
-    const main_image = post_data?.main_image?.imageFile
-    const main_video = post_data?.main_video?.imageFile?.publicURL
+const convertDate = (date) => {
+    const newdate = new Date(date)
+    return (
+        newdate.toLocaleString('en', { day: 'numeric' }) +
+        ' ' +
+        newdate.toLocaleString('en', { month: 'long' }) +
+        ' ' +
+        newdate.toLocaleString('en', { year: 'numeric' })
+    )
+}
+
+const ArticlesTemplate = (props) => {
+    const pathname = props.pageContext.pathname
+    const post_data = props.data.directus.blog[0]
+    const footer_banner_data = post_data?.footer_banners
+    const side_banner_data = post_data?.side_banners
+
+    const side_banner_data_details = {
+        max_w_value: '328px',
+        max_w_tablet: '320px',
+        isExternal: true,
+        redirectLink: side_banner_data?.cta_url,
+        imgSrcDesktop: side_banner_data?.banner_image?.imageFile,
+    }
+
+    const footer_banner_details = {
+        max_w_value: '792px',
+        max_w_tablet: '580px',
+        isExternal: true,
+        redirectLink: footer_banner_data?.cta_url,
+        imgSrcDesktop: footer_banner_data?.desktop_banner_image?.imageFile,
+        imgSrcMobile: footer_banner_data?.mobile_banner_image?.imageFile,
+    }
+
     return (
         <Layout>
             <SEO
-                description={localize('Single Article Post | Deriv')}
-                title={post_data.article_title}
+                description={
+                    'Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, and stock indices.'
+                }
+                title={'Markets | Markets to trade | Deriv'}
                 no_index
```

Comment:
```suggestion
                description={
                    'Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, and stock indices.'
                }
                title={'Markets | Markets to trade | Deriv'}
```

Reply:
We can remove `no_index` here as we need the title and description for these pages. We can pull the both of these from graphql.

Reply:
amended in latest commit